src/REPLMode/REPLMode.jl

```diff
@@ -652,9 +652,20 @@ Some commands have an alias, indicated below.
 end
 
 const help = gen_help()
+const REG_WARNED = Ref{Bool}(false)
 
 function try_prompt_pkg_add(pkgs::Vector{Symbol})
     ctx = Context()
+    if isempty(ctx.registries)
+        if !REG_WARNED[]
+            printstyled(ctx.io, " │ "; color=:green)
+            printstyled(ctx.io, "Attempted to search for missing packages in Pkg registries but no registries are installed.\n")
+            printstyled(ctx.io, " └ "; color=:green)
+            printstyled(ctx.io, "Use Pkg mode to install a registry. Both `pkg> add` and `pkg> update` will install the default registries.\n\n")
```

Comment:
Would be better to say something like
```
Run `using Pkg; Pkg.Registry.add()` to install default package registries.
```

Reply:
But that implies that the way to install default registries is `using Pkg; Pkg.Registry.add()`, but it's usually seamlessly managed by `pkg> add/update` etc. It feels odd to direct these people to the `using Pkg; Pkg...` style of interaction.

Also, it's worth noting that this only happens in interactive mode, where the Pkg repl is available. 
AFAICT the reason to give the `import Pkg; Pkg..` style action, like package loading errors do, is to cover when the pkg repl mode isn't available. i.e. in IJulia, which this user flow doesn't include

Reply:
Okay, suggest `pkg> registry add` then? `pkg> add<Enter>` and `pkg> update<Enter>` installs the default registries I think?

Reply:
If we want to give the user the "most minimal" command for downloading the default registries, how about we suggest that they do `import Pkg; Pkg.status()`.

`Pkg.status()` will download the default registries if no registries are available. However, unlike both `Pkg.add` and `Pkg.update`, `Pkg.status()` will make no modifications to the active environment. In contrast, both `Pkg.add` and `Pkg.update` can make changes to the active environment, which might not be something a user wants if they only want to download the available registries.

Reply:
And even if it did, why would you suggest `Pkg.status()` as a way to install registries when there is an actual command for installing registries `Pkg.Registry.add()`?

Reply:
I thought Ian didn't want to recommend `Pkg.Registry.add()`?

Reply:
I just tested it in a fresh depot, and `Pkg.status()` definitely downloads the registries.

I agree that `Pkg.Registry.add()` is the best thing to recommend. I was just under the impression that Ian specifically didn't want to recommend `Pkg.Registry.add()`, and that we needed somerhing else.

Reply:
> I just tested it in a fresh depot, and Pkg.status() definitely downloads the registries.

Okay this is new in 1.7 then. Why?

Reply:
Ahhh, was the debate just about recommending `import Pkg; Pkg.Registry.add()` versus recommending `] registry add`?

I don't really feel strongly between the two of those.

Reply:
```suggestion
printstyled(ctx.io, "Use Pkg mode to install a registry. In the Pkg REPL mode, `pkg> registry add` will install the default registries.\n\n")
```

Reply:
> Okay this is new in 1.7 then. Why?

Yeah it must be a new feature. I'm not sure why it was added.

Anyway, I went back through this discussion, and I realize I misunderstood the issue with `Pkg.Registry.add()` - it was just that we wanted to recommend the REPL mode instead.